src/it/resources/org/checkstyle/suppressionxpathfilter/lambdaparametername/SuppressionXpathRegressionLambdaParameterName2.java

```diff
@@ -0,0 +1,9 @@
+package org.checkstyle.suppressionxpathfilter.lambdaparametername;
+
+import java.util.function.Function;
+
+public class SuppressionXpathRegressionLambdaParameterName2 {
+    void test() {
+        Function<String, String> trimmer = s -> s.trim(); // warn
```

Comment:
Inputs between the 2 should be more unique.

Please add a parenthesis around the lambda parameter like https://github.com/checkstyle/checkstyle/blob/3835575706d8240b3ec054d17ed48f692b168611/src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/lambdaparametername/InputLambdaParameterName.java#L10 /

Reply:
OK,